### PR TITLE
feature: add command does not have to input all fields

### PR DESCRIPTION
**Pull request by xhsien**
@W10-B3/developers Please review.

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Person.java [hunk 1]
```diff
@@ -29,7 +29,7 @@
      * Every field must be present and not null.
```
Comment (Jemereny) [inline src/main/java/seedu/address/model/person/Person.java]: @chrisboo
Should change the comment here as you've edited the code

**Comment by Jemereny** [comment]
Changes look fine but you may want to edit the comments when you change the function of the method. Also, I think we should update the doc/help as we go along before we forget.

**Comment by xhsien** [comment]
@Jemereny I've updated the necessary docs.

**Comment by Jemereny** [comment]
Looks alright to me so far.

**Pull request merged**